test/e2e/suites/import-gitops-v3/import_gitops_v3_test.go

```diff
@@ -486,7 +484,7 @@ var _ = Describe("[GCP] [GKE] Create and delete CAPI cluster functionality shoul
 	})
 })
 
-var _ = Describe("[vSphere] [Kubeadm] Create and delete CAPI cluster from cluster class", Label(e2e.VsphereTestLabel, e2e.KubeadmTestLabel), func() {
+var _ = Describe("[vSphere] [Kubeadm] Create and delete CAPI cluster from cluster class", Ordered, Label("skip"), func() {
```

Comment:
do we skip it for now?

Reply:
resolving this should merge PR automatically